server/tests/route.test.ts

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-prototype-builtins */
+/* eslint-disable no-undef */
+import request from 'supertest';
+import connection from '../database/config/connection';
+import app from '../app';
+import buildFakeData from '../database/fakeData/buildFakeData';
+
+beforeAll(() => buildFakeData());
+
+describe('POST/login', () => {
+  test('User with admin role', async () => {
+    const response = await request(app)
+      .post('/api/login')
+      .send({
+        email: 'admin@gmail.com',
+        password: '123456789',
+      })
+      .expect(200);
+    expect(response.body.data.isAdmin).toEqual(true);
+    expect(response.body.data.name).toEqual('admin');
+    expect(response.body.data.hasOwnProperty('id')).toEqual(true);
+    expect(response.body.message).toBe('Successfully logged in');
+  });
+
+  test('User without admin role (Normal User)', async () => {
+    const response = await request(app)
+      .post('/api/login')
+      .send({
+        email: 'hosam@gmail.com',
+        password: '123456789',
+      })
+      .expect(200);
+    expect(response.body.data.hasOwnProperty('id')).toEqual(true);
+    expect(response.body.data.isAdmin).toEqual(false);
+    expect(response.body.message).toBe('Successfully logged in');
+  });
+
+  test('If is there access token', async () => {
+    const response = await request(app)
+      .post('/api/login')
+      .send({
+        email: 'admin@gmail.com',
+        password: '123456789',
+      })
+      .expect(200);
+    expect(
+      response.headers['set-cookie'][0].startsWith('ACCESS_TOKEN'),
+    ).toEqual(true);
+  });
```

Comment:
This is not a new case to test, you should test it with the previous two tests. if they set a cookie or not.

Reply:
Repaired.